### ISSUE TITLE
Link underline style

### ISSUE THREAD

**Issue by pynej** [papercss/papercss#155]
I'm not really sure the link underline goes well with the theme look and feel. I am thinking more of a single squiggly underline would be better. I don't have the skill alter the background-image gradient though through.

**Comment by rhyneav** [comment]
I hear you, a revamped design on the link styles would be well received! Something that better matches the button styles would look good.

**Comment by afzalsayed96** [comment]
Let me take a look at this

**Comment by serotonyn** [comment]
I would like to contribute and make my first Pr. I cloned the project to contribute on this issue, but then I saw some tools that i'm not familiar with like jade and gulp. Do you thing that I have to do some research on those before going any further ?

**Comment by TotomInc** [comment]
Hi @serotonyn and thanks for taking the time to contribute!

You don't really need to do some research on gulp and jade, they are tools we use to automate tasks and run tests, in other words this simplify the development of the framework (we already configured them) without the fear of breaking something.

However, these tools are very common in web-projects and I recommend you to take a look at what they do and how to setup them.

You can follow the [Contribution guide on the readme](https://github.com/papercss/papercss#contributing) which explains how to development environment.

**Comment by serotonyn** [comment]
Hi @TotomInc. I appreciate your welcoming attitude :)

Do you think this could be acceptable for matching the button style ?

![image](https://user-images.githubusercontent.com/22910212/48472377-6eb31000-e7f6-11e8-958b-f2a7b8f041b0.png)

And does it have to be a background image with a linear gradient ? The demo is done using an after pseudo element.

**Comment by TotomInc** [comment]
This looks great in the PaperCSS style, what do you think @rhyneav?

I think you need it to be an `:after` pseudo-element to make it entirely customizable (like a `div`). So in our case, we remove the default `text-decoration` and instead apply a `:after` pseudo-element.

**Comment by serotonyn** [comment]
Exactly, so i'm going to read through the contributing.md and prepare my first PR ! I'm so excited 😅

**Comment by rhyneav** [comment]
Wow, that looks awesome! Good stuff @serotonyn
